exchanges/hitbtc/hitbtc_websocket.go

```diff
@@ -21,6 +21,8 @@ const (
 	rpcVersion             = "2.0"
 )
 
+var requestID int64
```

Comment:
This might need to be protected due to a potential race for concurrent requests. I would feel more comfortable either atomic addition or mtx.